go/runtime/txpool/queues.go

```diff
@@ -50,6 +50,8 @@ type UsableTransactionSource interface {
 	// HandleTxsUsed is a callback to indicate that the scheduler is done with a set of txs, by hash. For most
 	// implementations, remove it from internal storage.
 	HandleTxsUsed(hashes []hash.Hash)
+	// PeekAll returns all transactions without removing them.
+	PeekAll() []*TxQueueMeta
```

Comment:
Why don't you name this `GetTxs` and move it after/before `GetTxByHash`?

Reply:
Because the operation is basically `TakeAll` without removing anything from the queues :D
`Get` is too vague -- does it remove txs from the queue or not? :)

Reply:
`GetTxsToPublish`, `GetTxByHash` and `GetTxs` (in txpool.go) also do not remove txs, so maybe its not that vague :)